cpp/tests/utilities/cudf_test_utils.cu

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2018 BlazingDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "cudf_test_utils.cuh"
+
+void print_gdf_column(gdf_column const * the_column)
+{
+  const size_t num_rows = the_column->size;
+  const gdf_dtype gdf_col_type = the_column->dtype;
+  switch(gdf_col_type)
```

Comment:
While you are moving these to a source file, would you mind modifying to use `type_dispatcher` to eliminate this switch and code duplication? Or if you don't have time, would you mind adding it to the list in https://github.com/rapidsai/cudf/issues/393